builtin/providers/aws/resource_aws_elastic_beanstalk_configuration_template.go

```diff
@@ -0,0 +1,259 @@
+// +build ignore
+
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elasticbeanstalk"
+)
+
+func resourceAwsElasticBeanstalkConfigurationTemplate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsElasticBeanstalkApplicationCreate,
+		Read:   resourceAwsElasticBeanstalkApplicationRead,
+		Update: resourceAwsElasticBeanstalkApplicationUpdate,
+		Delete: resourceAwsElasticBeanstalkApplicationDelete,
```

Comment:
This wiring looks off. Shouldn't these be `resourceAwsElasticBeanstalkConfigurationTemplate*`?

Also `elastic_beanstalk_configuration_template` didn't make it up into the provider resource list - so perhaps this is just dead code from an earlier revision?